### ISSUE TITLE
face detection / recognition

### ISSUE THREAD

**Issue by lanthale** [lessthanoptimal/BoofCV#214]
Is there a face detection algorithm already implemented ?

I am struggeling a bit what I should use:
- Openimaj: easy to use but, 350 MB of libs
- OpenCV: more difficult to use, 150 MB of libs
- BoofCV: Lightweight and fast and therefore my prefered one

But I cannot find anything about face detection which is my primary goal in my Photoslide application

**Comment by lessthanoptimal** [comment]
yeah was wondering about that from your comments before. Unfortunately there is still no face detector in BoofCV. It's been a glaring omission in BoofCV for a long time. Maybe this will be the year it gets added.

Maybe make it an optional feature and download as needed for now from one of those libs?

I've actually been considering writing an app similar to what you are doing. Maybe we could have a discussion in your project for how BoofCV could potentially add to that project?

**Comment by lanthale** [comment]
I have now started a discussion in my project and invited you.
